solr-synonyms-api/synonyms/constants/__init__.py

```diff
@@ -57,7 +57,7 @@ class EntityTypes(AbstractEnum):
     COOPERATIVE = 'CP'
     BENEFIT_COMPANY = 'BC'
     COMMUNITY_CONTRIBUTION_COMPANY = 'CC'
-    SOCIETY = 'SO'
+    SOCIETY = 'SOC'
```

Comment:
SHould this list of entity types match the one in namx-api?  If so, can you not import from namex-api rather than duplicate it in synonyms?